ts/features/wallet/component/NewMethodAddedNotifier.tsx

```diff
@@ -18,9 +18,9 @@ const newBancomatBottomSheet = () => {
 
   const openModalBox = async () => {
     const bottomSheetProps = await bottomSheetContent(
-      <BancomatInformation onAddPaymentMethod={dismiss} />,
+      <BancomatInformation onAddPaymentMethod={dismiss} hideInfobox={true} />,
       I18n.t("wallet.methods.pagobancomat.name"),
-      550,
+      385,
```

Comment:
```suggestion
      hideInfobox ? 385 : 550,
```
Could it be an improvement to avoid content scrolling when hideInfobox is false?

Reply:
I don't think it could be an improvement, the conditional visualization of the infobox in the content component is handled at the same component level of the bottom sheet definition so it wouldn't never change dinamically